contracts/scripts/createGenesis.js

```diff
@@ -0,0 +1,50 @@
+const ethers = require('ethers');
+const fs = require('fs');
+const faucet = require("../artifacts/src/pre-deploy/Faucet.sol/Faucet.json");
+const genesisPath = "../../clients/geth/specular/data/genesis.json";
```

Comment:
we should be able to pass a path in through a flag argument to the script, something like `--in base_genesis.json --out genesis.json`

also: snake case for filename, not camel